expression/bench_test.go

```diff
@@ -472,6 +472,15 @@ func (g *randDurInt) gen() interface{} {
 	return int64(rand.Intn(types.TimeMaxHour)*10000 + rand.Intn(60)*100 + rand.Intn(60))
 }
 
+type randJSONDurationWithLen struct{}
```

Comment:
Because it's not a common generator, could we put it in builtin_cast_vec_test.go?

Reply:
And we may change `randJSONDurationWithLen` to `randJSONDuration`.

Reply:
@tsthght Friendly ping, please address this comment.

Reply:
I had modified. PTAL @SunRunAway @qw4990